pkg/osquery/tables/dataflattentable/exec_and_parse.go

```diff
@@ -0,0 +1,101 @@
+package dataflattentable
+
+import (
+	"context"
+	"os"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/dataflatten"
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	"github.com/osquery/osquery-go/plugin/table"
+	"github.com/pkg/errors"
+)
+
+type flattenBytesInt interface {
+	FlattenBytes([]byte, ...dataflatten.FlattenOpts) ([]dataflatten.Row, error)
+}
+
+// execTableV2 is the next iteration of the dataflattentable wrapper. Aim to migrate exec based tables to this.
+type execTableV2 struct {
```

Comment:
I could do this using the existing table struct, as I did in #900, but it feels better to make a new struct and think about trying to clean that up.